src/kepler_model/cmd/cmd_util.py

```diff
@@ -354,6 +354,7 @@ def get_pipeline(
     energy_sources,
     valid_feature_groups,
     replace_node_type=default_node_type,
+    use_vm_metrics=False,
```

Comment:
I think we should allow passing labels to be selected instead. 
E.g. `kepler-model train --labels='job="vm"' --labels='foo="bar"` 
and keep a default of none, i.e single kepler instance.

Reply:
@KaiyiLiu1234 could you address this in a separate PR ?